content/_changelogs/12.1.0.md

```diff
@@ -0,0 +1,8 @@
+## 12.1.0
+
+_Released 12/20/2022_
+
+**Features:**
+
+- Added `cy.getAllCookies()` and `cy.clearAllCookies()`, which get and clear
+  cookies for all browser cookies across any domains
```

Comment:
this need the issue links as well
```suggestion
  cookies for all browser cookies across any domains.
  
**Bugfixes:**

- Fixed an issue with the [`cy.session()`](/api/commands/session) command where it now recollects the session data after validation is successful to correctly store the full browser context before the command ends. Fixes [#25112](https://github.com/cypress-io/cypress/pull/25112).
```

Reply:
Updated: [`030a688` (#4920)](https://github.com/cypress-io/cypress-documentation/pull/4920/commits/030a6881f38cc58b940917b6d575871e3a893d85)